src/evo/simplifiedmns.h

```diff
@@ -32,6 +33,8 @@ class CSimplifiedMNListEntry
     CBLSLazyPublicKey pubKeyOperator;
     CKeyID keyIDVoting;
     bool isValid{false};
+    uint16_t nType{CDeterministicMN::TYPE_REGULAR_MASTERNODE};
+    uint16_t platformHTTPPort{0};
```

Comment:
why only the http?

Reply:
@PastaPastaPasta As discussed with Platform team, only platformHTTPPort is included in SML entry.

Reply:
@ogabrielides @PastaPastaPasta 

What about `platformNodeID` and `platformP2PPort` for the mobile clients?

Mobile clients use P2P for connectivity.  Additionally, we also have some support for masternode keys in our apps (iOS for now) and having the `platformNodeID` will also be useful.

Reply:
Nevermind about `platformP2PPort` as the mobile clients will not be using this.